code/modules/research/xenobiology/crossbreeding/chilling.dm

```diff
@@ -0,0 +1,634 @@
+	w_class = WEIGHT_CLASS_HUGE
+	force = 5
+	max_charges = 1 //Recharging costs blood.
+	recharge_rate = 1
+	ammo_type = /obj/item/ammo_casing/magic/bloodchill
+	fire_sound = 'sound/effects/attackblob.ogg'
+
+/obj/item/gun/magic/bloodchill/process()
+	charge_tick++
+	if(charge_tick < recharge_rate || charges >= max_charges)
+		return 0
+	charge_tick = 0
+	var/mob/living/M = loc
+	if(istype(M) && M.blood_volume >= 20)
+		charges++
+		M.blood_volume -= 20
+	if(charges == 1)
+		recharge_newshot()
+	return 1
+
+/obj/item/ammo_casing/magic/bloodchill
+	projectile_type = /obj/item/projectile/magic/bloodchill
+
+/obj/item/projectile/magic/bloodchill
+	name = "blood ball"
+	icon_state = "pulse0_bl"
+	damage = 0
+	damage_type = OXY
+	nodamage = 1
+	hitsound = 'sound/effects/splat.ogg'
+
+/obj/item/projectile/magic/bloodchill/on_hit(mob/living/carbon/human/target)
+	. = ..()
+	if(ishuman(target))
+		target.apply_status_effect(/datum/status_effect/bloodchill)
+
+/mob/living/simple_animal/pet/dog/corgi/puppy/slime
+	name = "\improper slime corgi puppy"
+	real_name = "slime corgi puppy"
+	desc = "An unbearably cute pink slime corgi puppy."
+	icon_state = "slime_puppy"
+	icon_living = "slime_puppy"
+	icon_dead = "slime_puppy_dead"
+	nofur = TRUE
+	gold_core_spawnable = NO_SPAWN
+	speak_emote = list("blorbles", "bubbles", "borks")
+	emote_hear = list("bubbles!", "splorts.", "splops!")
+	emote_see = list("gets goop everywhere.", "flops.", "jiggles!")
+
+/obj/item/capturedevice
+	name = "gold capture device"
+	desc = "Bluespace technology packed into a roughly egg-shaped device, used to store nonhuman creatures. Can't catch them all, though - it only fits one."
+	w_class = WEIGHT_CLASS_SMALL
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "capturedevice"
+
+/obj/item/capturedevice/attack(mob/living/M, mob/user)
+	if(contents.len)
+		to_chat(user, "<span class='warning'>The device already has something inside.</span>")
+	if(!isanimal(M))
+		to_chat(user, "<span class='warning'>The capture device only works on simple creatures.</span>")
+		return
+	if(M.mind)
+		to_chat(user, "<span class='notice'>You offer the device to [M].</span>")
+		if(alert(M, "Would you like to enter [user]'s capture device?", "Yes", "No") == "Yes")
+			if(user.canUseTopic(src, BE_CLOSE) && user.canUseTopic(M, BE_CLOSE))
+				to_chat(user, "<span class='notice'>You store [M] in the capture device.</span>")
+				to_chat(M, "<span class='notice'>The world warps around you, and you're suddenly in an endless void, with a window to the outside floating in front of you.</span>")
+				store(M, user)
+			else
+				to_chat(user, "<span class='warning'>You were too far away from [M].</span>")
+				to_chat(M, "<span class='warning'>You were too far away from [user].</span>")
+		else
+			to_chat(user, "<span class='warning'>[M] refused to enter the device.</span>")
+			return
+	else
+		if(istype(M, /mob/living/simple_animal/hostile) && !("neutral" in M.faction))
```

Comment:
cant you make it check for factions between the mob and the user instead? xenobio can spawn mobs with the slime faction

Reply:
I could, but because regular gold cores only spawn with the neutral faction, it may be a bit restrictive.

Reply:
luminol race + gold extract + harm intent makes 1 mob with slime faction there may be other edge cases also slime pokeball